scrapers/VrxDB.yml

```diff
@@ -0,0 +1,27 @@
+name: VrxDB
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - vrxdb.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="elementor-widget-container"]//@title
```

Comment:
doesnt work in the test scene i pasted for the image
this  `//h1[contains(@class,"elementor-heading-title")]` should work ?